ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -336,6 +337,10 @@ instance TPraosCrypto c => QueryLedger (ShelleyBlock c) where
       :: Query (ShelleyBlock c) result
       -> Query (ShelleyBlock c) (Serialised result)
 
+    GetFilteredRegisteredStakeCreds
+      :: Set (SL.Credential 'SL.Staking c)
+      -> Query (ShelleyBlock c) (SL.StakeCreds c)
```

Comment:
I'm double checking, but I think this one will not be needed, since the `StakeCreds` does not contain any useful information, and indeed will soon be changed to be just a `Set` once we remove decaying rewards.

Reply:
indeed!